go-client/go-test-to-junit.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eux;
```

Comment:
I would start:

```
#!/usr/bin/env bash

set -o errexit
set -o pipefail
set -o nounset
set -o xtrace
...
```

Reply:
Has this convention been updated elsewhere, documented so we can be sure to fix existing ones, future ones?

...and if we're changing the standard again, is it worth changing it further?